x-pack/solutions/security/plugins/security_solution/public/threat_intelligence/modules/indicators/components/flyout/block.tsx

```diff
@@ -56,24 +61,25 @@ export const IndicatorBlock: FC<IndicatorBlockProps> = ({
   indicator,
   'data-test-subj': dataTestSubj,
 }) => {
+  const { key, value } = getIndicatorFieldAndValue(indicator, field);
+
   return (
-    <EuiPanel {...panelProps}>
-      <VisibleOnHover data-test-subj={`${dataTestSubj}Item`}>
-        <EuiText>
-          <IndicatorFieldLabel field={field} />
-        </EuiText>
-        <EuiSpacer size="s" />
-        <EuiText size="s">
-          <IndicatorFieldValue indicator={indicator} field={field} />
-          <span className="actionsWrapper">
-            <IndicatorValueActions
-              indicator={indicator}
-              field={field}
-              data-test-subj={dataTestSubj}
-            />
-          </span>
-        </EuiText>
-      </VisibleOnHover>
-    </EuiPanel>
+    <SecurityCellActions
+      data={{ field: key, value }}
+      mode={CellActionsMode.HOVER_DOWN}
+      triggerId={SecurityCellActionsTrigger.DEFAULT}
+    >
+      <EuiPanel {...panelProps}>
+        <VisibleOnHover data-test-subj={`${dataTestSubj}Item`}>
```

Comment:
I think this `VisibleOnHover` is not needed anymore no? I removed it and didn't see any differences. It might have a slight impact on Cypress tests but I'm hoping it won't be hard to solve